demo/actions.py

```diff
@@ -485,3 +487,86 @@ def submit(self,
         dispatcher.utter_template('utter_restart_with_button', tracker)
 
         return [FollowupAction('action_listen')]
+
+
+class CommunityEventAction(Action):
+    """Utters Rasa community events."""
+
+    def __init__(self):
+        self.last_event_update = None
+        self.events = None
+        self.events = self._get_events()
+
+    def name(self) -> Text:
+        return "action_get_community_events"
+
+    def _get_events(self) -> List['CommunityEvent']:
+        if self.events is None or self._are_events_expired():
```

Comment:
hmm, do we need any of these checks? because every time you call the action, self.events will be None :P

Reply:
Are the action objects re-created between the requests? When I check the logs of the action endpoint it actually just calls `_get_events` when the server is started or the events are expired

Reply:
Hm yes actually good point, should be fine